suite-common/icons/generateIconFont.ts

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable no-console */
+// disable eslint import for this file because this whole file is using only devDependencies
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { generateFonts, FontAssetType, OtherAssetType } from 'fantasticon';
+import fs, { constants } from 'fs';
+import path from 'path';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import chalk from 'chalk';
+
+import { type IconName as AllAvailableIcons } from './src/icons';
+import { MOBILE_ICON_FONT_NAME } from './src/constants';
+
+const usedIcons = [
+    'arrowDown',
+    'arrowDown',
+    'arrowLineDown',
+    'arrowLineUp',
+    'arrowLineUpRight',
+    'arrowRight',
+    'arrowsCounterClockwise',
+    'arrowSquareOut',
+    'arrowUp',
+    'arrowUpRight',
+    'arrowUpRight',
+    'arrowURightDown',
+    'backspace',
+    'bookmark',
+    'bugBeetle',
+    'calendar',
+    'caretCircleRight',
+    'caretDown',
+    'caretDown',
+    'caretDownFilled',
+    'caretLeft',
+    'caretRight',
+    'caretUp',
+    'caretUpDown',
+    'caretUpFilled',
+    'change',
+    'check',
+    'checkCircle',
+    'checkCircleFilled',
+    'checks',
+    'circleDashed',
+    'clockClockwise',
+    'coins',
+    'coinVerticalCheck',
+    'copy',
+    'database',
+    'detective',
+    'discover',
+    'eye',
+    'eyeSlash',
+    'facebookLogo',
+    'filePdf',
+    'fingerprint',
+    'fingerprintSimple',
+    'flag',
+    'flagCheckered',
+    'gear',
+    'githubLogo',
+    'house',
+    'image',
+    'info',
+    'lifebuoy',
+    'lightbulb',
+    'link',
+    'lock',
+    'magnifyingGlass',
+    'palette',
+    'password',
+    'pencil',
+    'pencilSimpleLine',
+    'piggyBank',
+    'piggyBankFilled',
+    'plus',
+    'plusCircle',
+    'qrCode',
+    'question',
+    'question',
+    'question',
+    'question',
+    'shareNetwork',
+    'shieldWarning',
+    'shuffle',
+    'stack',
+    'swap',
+    'trashSimple',
+    'treeStructure',
+    'trezorLogo',
+    'trezorModelOne',
+    'trezorModelOneFilled',
+    'trezorModelT',
+    'trezorModelTFilled',
+    'trezorSafe3',
```

Comment:
Icon for Safe 3 is somehow broken 
<img width="482" alt="image" src="https://github.com/user-attachments/assets/78a634df-b6df-4da0-a27c-28de7bb98191">

correct:
<img width="389" alt="image" src="https://github.com/user-attachments/assets/1cb7a457-30f0-46a4-8773-20967aebc765">

Reply:
There could be sometimes problem when svg uses `fill-rule` attribute but I verified other icons we use that have this property and they look correct.

I fixed this icon manually and added guide how to fix it to README in case we will need it.

Reply:
Looks good now 👍 
<img width="369" alt="image" src="https://github.com/user-attachments/assets/d245017c-5e5a-4571-afd1-cdeee3732f6f">
<img width="371" alt="image" src="https://github.com/user-attachments/assets/d604370a-2808-49c7-aae4-66f670988555">